berkeley-mobile/StudyPact/ProfileViewController.swift

```diff
@@ -205,13 +272,15 @@ extension ProfileViewController {
         facebookLabel.adjustsFontSizeToFitWidth = true
         facebookLabel.minimumScaleFactor = 0.7
```

Comment:
change this to say Facebook username and have another label with facebook.com/ prior to the text field.